projects/plugins/jetpack/_inc/lib/class-jetpack-recommendations.php

```diff
@@ -219,7 +219,7 @@ public static function plugin_activated( $plugin ) {
 
 			// Check for a plan or product that enables scan.
 			$plan_supports_scan = Jetpack_Plan::supports( 'scan' );
-			$has_scan_product   = count( array_intersect( array( 'jetpack_scan', 'jetpack_scan_monthly' ), $products ) ) > 0;
+			$has_scan_product   = array_intersect( array( 'jetpack_scan', 'jetpack_scan_monthly' ), $products ) !== array();
```

Comment:
In this case, it would be fine. `$products` is set by `array_column' which would return an empty array. If further up the stack, `null` was passed, it would have fataled by now.